src/utils/SortOrders.ts

```diff
@@ -0,0 +1,5 @@
+export const enum SortOrders {
+  DEFAULT = '',
+  DESC = 'desc',
+  ASC = 'asc',
+}
```

Comment:
You actually don't need `asc` type of order. If you see the example, the order is only shown with descending value